package/MDAnalysis/analysis/utils/prog.py

```diff
@@ -0,0 +1,102 @@
+""" Add docstring later
```

Comment:
Done, moved code to `lib.log` and also modified the bar with threading so that it supports working with serial jobs. Hopefully that's what you meant.